tools/bouncer_worker/src/tasks/unityEditor.js

```diff
@@ -75,9 +83,16 @@ UnityHandler.generateAssetBundles = async (database, model, rid, logDir, process
 		return retVal;
 	} catch (err) {
 		logger.info(`Failed to execute unity command: ${err}`, logLabel);
-		const invalidLicence = err === ERRCODE_UNITY_LICENCE_INVALID || await checkLicenceError(unityLog);
-		throw invalidLicence ? ERRCODE_UNITY_LICENCE_INVALID : ERRCODE_BUNDLE_GEN_FAIL;
-	}
+		switch (err) {
+			case ERRCODE_UNITY_LICENCE_INVALID:
+			case ERRCODE_REPO_LICENCE_INVALID:
+				throw err;
+			default:
+				throw await checkLicenceError(unityLog) ? ERRCODE_UNITY_LICENCE_INVALID : ERRCODE_BUNDLE_GEN_FAIL;
+		}
+	} finally{
+    delete process.env.REPO_INSTANCE_ID
```

Comment:
There isn't much reason to do this.

it'll get overwritten eveytime something is ran.

if we do this then technically we should be doing this for every env var we're setting.